.github/workflows/main.yml

```diff
@@ -26,13 +26,11 @@ jobs:
           auto-update-conda: true
           channels: conda-forge
 
-      - name: Install dependencies & build
+      - name: Install dependencies
         shell: bash -l {0}
         run: |
           conda activate test
-          
           conda install --yes -c conda-forge compilers gfortran
```

Comment:
It may be better to use homebrew and do `brew install gfortran` on the macOS runners